### ISSUE TITLE
Image preview doesn't work with newest versions of urxvt

### ISSUE THREAD

**Issue by hektr** [ranger/ranger#2381]
Something is broken in newest versions of urxvt. Ranger image previewing works only with urxvt v.9.22 or may be with older versions, but is not working with version 9.25 and 9.26

**Comment by toonn** [comment]
Are you sure you have a urxvt with pixbuf support?

**Comment by hektr** [comment]
Yes, i tested first of all with enabled pixbuf. On newest versions it works only with w3m preview method.
You can read comments in aur [rxvt-unicode-pixbuf](https://aur.archlinux.org/packages/rxvt-unicode-pixbuf/)

**Comment by toonn** [comment]
I suspect this is a regression in urxvt.

**Comment by hektr** [comment]
Yes, I never liked urxvt, it's too ugly and unusable if you don't invest a lot of time on its customization. Terminology looks good and does have a lot of bells and whistles out of box. You can use it just after you install it. But ranger's image previewing doesn't work very good with terminology :(

**Comment by toonn** [comment]
What? The terminology preview method has been first-class in my experience. If that doesn't work try kitty, which also has a native image previewing method.

**Comment by hektr** [comment]
It shows previews, but it does have side effects. For example, it doesn't show command hint if you are on previewed image. And something strange happens, when you press `v` or `space` button to select files if image preview is enabled. [Example](https://youtu.be/qvAReA-KFus)

**Comment by toonn** [comment]
Those are definite bugs. Kitty has fewer problems fwiw. And there's a PR adding Sixel support which should add a decent method to more terminals. There's also ueberzug if you're running X11.

**Comment by hektr** [comment]
with ueberzug it works much better :)
